shremote.py

```diff
@@ -355,17 +380,23 @@ def get_log_dir(cls):
             return Config.instance().logs.dir
         elif 'log_dir' in Config.instance().programs:
             return Config.instance().programs.log_dir
+        elif 'log_dir' in Config.instance().dirs:
+            return Config.instance().dirs.log_dir
         else:
-            raise Exception("Cannot locate logs in cfg.logs.dir or cfg.programs.log_dir")
+            log_warn("Cannot locate logs in cfg.logs.dir, cfg.programs.log_dir or cfg.dirs.log_dir.\
+                      Using user home directory.")
+            return str(Path.home())
```

Comment:
Just realized that this will likely not work, as it assumes that the home directory is the same on all of the machines. Would prefer to just use `~` instead of Path.home(). The user may change across machines as well.

Reply:
What about 
```
            Config.instance().set_permanent(log_dir = '~/')
            return '~/'
```

So that it also becomes available in the top level of the config (not necessary but could be handy)